src/panels/lovelace/entity-rows/hui-input-select-entity-row.ts

```diff
@@ -51,25 +54,26 @@ class HuiInputSelectEntityRow extends LitElement implements EntityRow {
     }
 
     return html`
-      <state-badge .stateObj="${stateObj}"></state-badge>
-      <paper-dropdown-menu
-        selected-item-label="${stateObj.state}"
-        @selected-item-label-changed="${this._selectedChanged}"
-        label="${this._config.name || computeStateName(stateObj)}"
-      >
-        <paper-listbox
-          slot="dropdown-content"
-          selected="${stateObj.attributes.options.indexOf(stateObj.state)}"
+      <hui-generic-entity-row .hass="${this.hass}" .config="${this._config}">
```

Comment:
This is not the same. Now it will render the name besides the dropdown. This was not done before because the dropdown already had the name set as a label.

Reply:
lame...lol